src/JukeBox.cpp

```diff
@@ -68,9 +69,15 @@ TResult<TSessionID> JukeBox::generateSession(optional<TPassword> const &pw,
     LOG(INFO) << "JukeBox.generateSession: User '" << name << "' is admin!";
     user.isAdmin = true;
   }
+
+  /* Generate a unique ID, consisting of a counter and the number of seconds
+   * since 1970 */
+  user.SessionID = "ID" + userID + to_string(time(nullptr));
```

Comment:
i am a little surprised that `const char * + int + std::string` results in a string